backend/apps/ai/common/utils.py

```diff
@@ -4,29 +4,54 @@
 import time
 from datetime import UTC, datetime, timedelta
 
+import openai
+
 from apps.ai.common.constants import (
     DEFAULT_LAST_REQUEST_OFFSET_SECONDS,
     MIN_REQUEST_INTERVAL_SECONDS,
 )
 from apps.ai.models.chunk import Chunk
+from apps.ai.models.context import Context
 
 logger: logging.Logger = logging.getLogger(__name__)
 
 
+def create_context(content: str, content_object=None, source: str = "") -> Context:
+    """Create and save a Context instance independently.
+
+    Args:
+        content (str): The context content
+        content_object: Optional related object
+        source (str): Source identifier
+
+    Returns:
+        Context: Created Context instance
+
+    """
+    context = Context(content=content, content_object=content_object, source=source)
+    context.save()
+    return context
```

Comment:
I couldn't find a logic related to context updates -- only creation. It seems this approach create multiple contexts to the same entity while the idea is have a single source of truth for each entity context.

This should be part of `Context` model -- you already had similar issue to fix earlier. Please follow our regular way of handling model updates.